### ISSUE TITLE
train loss and validation loss both become 0 while training RNN LM model

### ISSUE THREAD

**Issue by Yibin-Lei** [speechbrain/speechbrain#1243]
I use my own dataset to train an RNN model using the RNNLM.yaml and train.py in directory speechbarin/templates/speechrecognition/LM, but when it comes to epoch 2, the train loss is "0" and the valid loss is also "0". Is it normal(model has converged) or there exists something wrong?

<img width="1390" alt="b2b24ebec38d7fd9bbd8e22cc2d25de" src="https://user-images.githubusercontent.com/60934253/148992126-8fa9d2fe-5d21-40f6-9942-3e8544f6571c.png">

**Comment by Gastron** [comment]
Most likely something is wrong in training. It could be an issue in the inputs and outputs of the model - maybe the language model can somehow see the output it needs to predict or something.

If the data is so small and easy that the model is able to completely memorize it and the validation set can be perfectly predicted using the training data, this could happen, but then the data is probably not really useful.

**Comment by mravanelli** [comment]
Correct. The task might be too easy (and thus solved perfectly) or there might be an overlap between training and validation data.

**Comment by Yibin-Lei** [comment]
I get it. I think maybe the task is too easy. Then given that the task is easy, do you think that adding LM for decoding may decrease the performance?

**Comment by Gastron** [comment]
Generally yes, unless that LM data is a good match for what you expect to recognize. But that is an empirical question, you'll know by testing.

**Comment by Yibin-Lei** [comment]
Thanks!
